src/components/JobModalNew.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import Modal from '@material-ui/core/Modal';
+import { withStyles } from '@material-ui/core/styles';
+import JobModalFields from './JobModalFields';
+
+const styles = theme => ({
+    paper: {
+        position: 'absolute',
+        width: theme.spacing.unit * 50,
+        backgroundColor: theme.palette.background.paper,
+        boxShadow: theme.shadows[5],
+        padding: theme.spacing.unit * 4,
+        outline: 'none',
+    },
+});
+
+class JobModalNew extends React.Component {
```

Comment:
this doesn't need to be a class and we don't need the `handleClose` method as we're defining similar in the `CardComponent`